src/app/telemetry/telemetry.component.html

```diff
@@ -48,6 +48,9 @@
             <mat-option *ngFor="let schema of editorService.schemaTypes" [value]="schema"> {{ schema }} </mat-option>
         </mat-select>
     </mat-form-field>
+    <ng-container *ngIf="isObjectSchema()">
+        <button mat-button (click)="openObjectSchemaEditor()">Edit</button>
```

Comment:
Looking to use mat icons as much as possible. Check this out:

<mat-icon>edit</mat-icon>

https://fonts.google.com/icons?selected=Material+Icons